### DISCUSSION TITLE
Visuals

### DISCUSSION THREAD

**Discussion by emiljapelt** [git-truck/git-truck#65] [Research]
How should we visualize a repo?

Some ideas are available [here](https://drive.google.com/file/d/1oiNCkQL8ThGJT9KIRLSgxcPYxB2_0CLD/view?usp=sharing). The drawings are viewable via draw.io / diagrams.net.

**Comment by tjomson** [comment]
Resolves #8

**Comment by hojelse** [comment]
Idea:

- 1 gradient from red to some neutral colour, according to percentage dominans. Answering the question "Which files have a high truck factor?".
- Author breakdown shown when hovering. Answering the question "Who is the dominant author".

![IMG_20220214_165455__01.jpg](https://user-images.githubusercontent.com/34659757/153898991-d7cdb63a-4818-4507-9bd1-229a9398c8f0.jpg)
